src/js/modules/CartMini.js

```diff
@@ -0,0 +1,148 @@
+import { cartTemplate } from "./cart-template.js";
+// import {
+//   getCartProducts,
+//   setCartValues,
+//   Services.updateCartProduct,
+// } from "../services/sharedproduct.service.js";
+import ProductServices from '../services/sharedproduct.service.js';
+
+class CartMini {
+  cartListElement = document.getElementById("cart-list");
+  contentHided = document.querySelectorAll(".js-hide-on-none");
+  numberInCart = document.querySelector(".js-number-cart");
+  numberInCartItem = document.querySelector(".js-number-cart-item");
+  addToCart = document.querySelectorAll(".js-add-cart");
+  constructor() {
+    this.cartPopUp();
+  }
+
+  cartPopUp() {
+    let cartDisplay = document.querySelector(".js-cart");
+    let cartButton = document.querySelector(".js-cart-open");
+
+    const self = this;
+    cartButton.addEventListener("click", function () {
```

Comment:
cart & mini cart will both load products ?

I would suggest a flow:
- UI calls to  Cart Service (share for both mini cart and cart)
- Cart Service calls Product service to get product info if needed
- Cart Service updates in-mem data model
- Cart service publishes CartChanged event to update UI
- CartComponent & MiniCartComponent subscribe to  CartChanged and modify DOM

Reply:
Separate to layers:
XXXComponents: deal with Dom
XXXServices: deal with biz, 1 responsibility at a time